app/webpacker/components/RegistrationsV2/RegistrationEdit/Payments.jsx

```diff
@@ -1,26 +1,27 @@
 import { useMutation, useQuery } from '@tanstack/react-query';
 import React, { useEffect } from 'react';
 import {
-  Button, Header, Message, Table,
+  Button, List, Message, Table,
 } from 'semantic-ui-react';
-import getAvailableRefunds from '../api/payment/get/getAvailableRefunds';
+import getRegistrationPayments from '../api/payment/get/getRegistrationPayments';
 import refundPayment from '../api/payment/get/refundPayment';
 import Loading from '../../Requests/Loading';
 import AutonumericField from '../../wca/FormBuilder/input/AutonumericField';
 import useInputState from '../../../lib/hooks/useInputState';
 import { useConfirm } from '../../../lib/providers/ConfirmProvider';
 import I18n from '../../../lib/i18n';
+import { isoMoneyToHumanReadable } from '../../../lib/helpers/money';
 
-export default function Refunds({
-  onSuccess, userId, competitionId,
+export default function Payments({
+  onSuccess, registrationId, competitionId,
 }) {
   const {
-    data: refunds,
-    isLoading: refundsLoading,
+    data: payments,
+    isLoading: paymentsLoading,
     refetch,
   } = useQuery({
-    queryKey: ['refunds', competitionId, userId],
-    queryFn: () => getAvailableRefunds(competitionId, userId),
+    queryKey: ['payments', registrationId],
+    queryFn: () => getRegistrationPayments(competitionId, registrationId),
     refetchOnWindowFocus: false,
     refetchOnReconnect: false,
     staleTime: Infinity,
```

Comment:
Technically unrelated but very, very easy fix: Aren't these options (and the fourth one below, which GitHub doesn't let me comment on because it arbitrarily rules that it's too far away from the changed lines, grr....) part of the default `WCAQueryClientProvider` setup these days?